test/runtime/samples/apply-directives-in-order-2/_config.js

```diff
@@ -1,7 +1,8 @@
-const value = [];
+let value = [];
```

Comment:
I had to do this get props() thing because of shared state between runs without hydration and with. 

I should probably make all these default exports a closure instead of this chicanery.